charts/policy-reporter/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: |
   It creates Prometheus Metrics and can send rule validation events to different targets like Loki, Elasticsearch, Slack or Discord
 
 type: application
-version: 2.9.3
+version: 2.9.4
 appVersion: 2.6.1
```

Comment:
kann you also prepare the new appVersion 2.6.2 here and in the values.yaml, which I will release when everything is merged

Reply:
Done